app/models/responsiblepeople/ResponsiblePerson.scala

```diff
@@ -312,7 +358,7 @@ object ResponsiblePerson {
 
   implicit class FilterUtils(people: Seq[ResponsiblePerson]) {
     def filterEmpty: Seq[ResponsiblePerson] = people.filterNot {
-      case _@ResponsiblePerson(None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, _, _, _, _, _, _) => true
+      case _@ResponsiblePerson(None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, ApprovalFlags(None, None), _, _, _, _, _, _) => true
```

Comment:
Don't think you need the None, None in ApprovalFlags

Reply:
Tried to remove however; compiler seems to think it does.

Reply:
ok